app/browser/tabs.js

```diff
@@ -674,6 +675,32 @@ const api = {
         }
       })
 
+      tab.on('before-autofill', (e, values) => {
+        tabMessageBox.show(tabId, {
+          message: values.join('\n'),
+          title: 'You are about to autofill:',
```

Comment:
this should be localized. you can use the`l10n.translation(STRING_HERE)` method and add that string to `common.properties` file to make it

Reply:
thanks for reminding.